design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,47 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task, it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review, but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview / Problem Statement
+
+* Describe the task with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals / In-Scope
+
+* List a few bullet points of what this task will achieve and that are most relevant for the design review discussion.
+* This should include acceptance criteria required to meet the [definition of done](../../team-agreements/definition-of-done/).
```

Comment:
I disagree, definition of done should be documented as part of the working agreement and apply to all work. I would not duplicate the documentation of that in other areas. I would remove this bullet

Reply:
Similar to my note above.  I don't think we should be duplicating data available from othe sources but also feel the design review should have enough information to review without having to reference other material so that it can easily be referenced at a later point in time after a project goes out of scope.

Reply:
I agree with @wbreza on the fact that there should be enough information on the document so it can be reviewed as an independent document without having to reference other material.